### ISSUE TITLE
PhotoSwipe only working for admin

### ISSUE THREAD

**Issue by FredFabLab** [Piwigo/piwigo-bootstrap-darkroom#356]
I have the following problem: I'd like to open PhotoSwipe's lightbox by default when clicking on a picture's thumbnail. I've found the corresponding setting under `Configuration > Themes > Boorstrop Darkroom > Configuration > Settings > Appearance > Thumbnail page display > Link thumbnail to PhotoSwipe Slideshow`. 
It works perfectly fine when I'm logged in as the webmaster. But for other users (`User` or `Generic`), the Picture details page opens, the full-screen button is also not working.
**Is there a requirement to enable PhotoSwipe for all users?** I didn't find an option for that… Or do you have any other ideas?

Tested on:
- Firefox 110.0.1 and Safari 16.3 on macOS Ventura 13.2.1
- Piwigo 13.6.0
- Bootstrap Darkroom 2.5.17
- PHP: 8.0.25

**Comment by FredFabLab** [comment]
Also thought so, but @MondyNor mentioned it's solved in 2.5.17 while I still have the problem within this version.

**Comment by MondyNor** [comment]
> Sounds like the same problem: #353

No, the problem i had was to show pictures full screen, as i understand his post he has problem with getting the slideshow to work. Anyway maybe it can be related to a plugin so here are a list of all the plugins i have installed and which ones i have active https://gyazo.com/33e30cf5a577a8947801709c3a321f7e

**Comment by MondyNor** [comment]
You could also use the watermark option for that, which i do :) Good to her you solved it

**Comment by DeWir3** [comment]
> 

Thanks for the script! I managed to disable the right-click menu with that using the Personal Plugin option as described here https://github.com/Piwigo/piwigo-bootstrap-darkroom/wiki/Add-custom-CSS-and-Javascript
